mmaction/core/evaluation/accuracy.py

```diff
@@ -356,12 +356,6 @@ def get_weighted_score(score_list, coeff_list):
     return weighted_scores
 
 
-def softmax(x, dim=1):
-    """Compute softmax values for each sets of scores in x."""
-    e_x = np.exp(x - np.max(x, axis=dim, keepdims=True))
-    return e_x / e_x.sum(axis=dim, keepdims=True)
-
-
```

Comment:
Why delete it?

Reply:
Ok, it seem it is the same with `from scipy.special import softmax`

Reply:
... seems scipy is not an allowed package, roll back this commit